### DISCUSSION TITLE
Automatic testing of UI created in Trame

### DISCUSSION THREAD

**Discussion by kdasVF** [Kitware/trame#411] [General]
Do you have any suggestions on how best to test an app UI created in Trame automatically? A quick Google search shows Selenium, Cypress, etc. Want to hear from someone who has experience using any one of them.

**Comment by jourdain** [comment]
We use Selenium from python. You can see some of that in the [trame-client repo](https://github.com/Kitware/trame-client/blob/master/tests/test_vue23.py).
